src/assets/app.sass

```diff
@@ -21,5 +21,9 @@ h1
 .sy-color
     color: $primary
 
+.remove-margin
+    margin-right: 0
+    margin-left: 0
+
```

Comment:
Könnte das nicht von Bootstraps hauseigener Klasse -> `mx-0` übernommen werden?
In eurem HTML wäre es dann `class="mx-0"` macht genau das Selbe.
Wenn ihr eine eigene Klasse dafür möchtet, welche eventuell ein bisschen sprechender ist dann 👍

Reply:
Ja, ich würde bei der hauseigene Klasse bleiben, ist an manchen Stellen etwas übersichtlicher.

Reply:
Die Klasse wurde nichts benutzt. Hab sie jetzt rausgeschmissen